lib/init.php

```diff
@@ -113,7 +113,7 @@ function gutenberg_site_editor_menu() {
  * @param WP_Admin_Bar $wp_admin_bar Core class used to implement the Toolbar API.
  */
 function modify_admin_bar( $wp_admin_bar ) {
-	if ( gutenberg_use_widgets_block_editor() && ! is_admin() ) {
+	if ( gutenberg_use_widgets_block_editor() && $wp_admin_bar->get_node( 'widgets' ) !== NULL ) {
```

Comment:
This change fails the PHPCS check. Otherwise, it looks good to me.

PHPCS error:
```
TRUE, FALSE and NULL must be lowercase; expected
"null" but found "NULL"
(Generic.PHP.LowerCaseConstant.Found)
```

Reply:
```suggestion
	if ( gutenberg_use_widgets_block_editor() && $wp_admin_bar->get_node( 'widgets' ) !== null ) {
```
Good catch. Thought I spotted capitals somewhere. My original suggestion was correctly formatted. :)

@JustinyAhin, you should be able to directly commit this suggestion.